frontend/packages/ceph-storage-plugin/src/features.ts

```diff
@@ -20,19 +23,41 @@ export const detectIndependentMode: ActionFeatureFlagDetector = (dispatch) =>
     },
   );
 
-export const OCS_VERSION_4_5_FLAG = 'OCS_VERSION_4_5_FLAG';
+export const OCS_SUPPORT_FLAGS = {
+  OCS_SNAPSHOT: 'OCS_SNAPSHOT',
+  // OCS_EXTERNAL: 'OCS_EXTERNAL'
+};
```

Comment:
This works only when we have one value in here. If you have multiple it will give you errors. See `reduces/features` for more details. As we are only allowed to call dispatch for flags which have already been defined via the extension point. However you are trying to set flags which are undefined previously. Right now it works since OCS_SNAPSHOT is being defined via the extension point. Try adding OCS_EXTERNAL in here and you should be getting an error.